packages/vue-remark/README.md

```diff
@@ -0,0 +1,151 @@
+# @gridsome/vue-remark
+
+> Create pages with Vue components in markdown. Perfect for building Documentation, Design Systems, Portfolios, Blogs, etc.
+
+## Install
+
+- `yarn add @gridsome/vue-remark`
+- `npm install @gridsome/vue-remark`
+
+## Usage
+
+```js
+module.exports = {
+  plugins: [
+    {
+      use: '@gridsome/vue-remark',
+      options: {
+        typeName: 'MarkdownPage', // required
+        baseDir: './src/pages', // required
+      }
+    }
+  ]
+}
+```
+
+## Example component
+
+```jsx
+---
+title: A cool title
+excerpt: Lorem Ipsum is simply dummy text.
+layout: ~/layouts/Default.vue
+---
+import YouTube from '~/components/YouTube.vue'
+import data from '~/data/youtube.json'
+
+# {{ $frontmatter.title }}
+
+<YouTube :id="data.id" />
+
+> {{ $frontmatter.excerpt }}
+```
+
+## Options
+
+#### typeName
+
+- Type: `string` *required*
+
+The type name to give the pages in the GraphQL schema.
+
+#### baseDir
+
+- Type: `string` *required*
+
+The path to the directory which contains all the `.md` files. A relative path will be resolved from the project root directory.
+
+#### component
+
+- Type: `string`
+
+Use a custom component as template for every page created by this plugin. This option is useful if you for example need to have a shared `page-query` or want to wrap every page in the same layout component. Insert the `VueRemarkContent` component where you want to show the Markdown content.
+
+```html
+<template>
+  <Layout>
+    <h1>{{ $page.vueRemarkPage.title }}</h1>
+    <VueRemarkContent />
+  </Layout>
+</template>
+
+<page-query>
```

Comment:
Is this actually working for templates and page queries?  I'm not able to get it working.  Looking at the code, I suspect it's not since you are only using the createPages api and not doing any of the operations the source-filesystem plugin does, although maybe I'm mistaken about what that accomplishes.

Reply:
To elaborate, `$page` is null for me, but I can do a static-query just fine.

Reply:
I'm not able to reproduce this one. Do you have `component: './src/templates/SomeComponent.vue'`, but `page-query` doesn't work in that component?

Reply:
Correct.  (And I'm not aware of any way to get any sort of debugging output regarding the query beyond that.)

Reply:
This works:
```
<static-query>
query {
  mytype (path: "/a-const-path") {
    path
    title
  }
}
```

This does not:
```
<page-query>
query {
  mytype (path: "/a-const-path") {
    path
    title
  }
}
```

And of course, using `page-query` with the $id or $path parameters also does not work for me

Reply:
I figured it out...I need ".vue" at the end of my component name.  Is that expected?

Reply:
Good :) Yes, it must include the extension in order to parse the file in Node.js.